server/models/meta.py

```diff
@@ -50,33 +50,33 @@ class TCGAModel(object):
     TCGAIndices = []
 
     #: Matches valid case names
-    case_re = re.compile('tcga-[a-z0-9]{2}-[a-z0-9]{4}', flags=re.I)
+    case_re = re.compile(r'tcga-[a-z0-9]{2}-[a-z0-9]{4}', flags=re.I)
 
     #: Matches valid uuid's
     uuid_re = re.compile(
-        '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}',
+        r'[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}',
         flags=re.I
     )
 
     #: Matches valid tcga barcodes
     barcode_re = re.compile(
-        case_re.pattern + '[0-9a-z-]*',
+        case_re.pattern + r'[0-9a-z-]*',
```

Comment:
Is it considered good practice to use raw strings even when the string doesn't contain a backslash?

Reply:
I thought it was considered good practice to use raw strings for regexs in general.